tests/stream.cc

```diff
@@ -16,26 +17,64 @@ class FakeDB : public Database {
   KVStore store_;
 };
 
+// Test class containing instance of mock DB
```

Comment:
Creating a class for one of the tests is stylistically out of line with the other tests.

Reply:
I'm confused. I copied the test class from the funcs.cc FakeDB and FuncsTest  where a mock db is also instantiated for testing. Or maybe should I just combine my tests in to that class?